driver/src/main.rs

```diff
@@ -184,6 +184,13 @@ struct Options {
         parse(try_from_str)
     )]
     use_shadowed_orderbook: bool,
+
+    #[structopt(
+        long,
+        env = "ORDERBOOK_FILEPATH",
+        default_value = "/var/db/stablex_orderbook.ron"
+    )]
+    orderbook_filepath: String,
```

Comment:
Let's make this a `PathBuf`. I found this in the structop documentation:
```rust
    /// Input file
    #[structopt(parse(from_os_str))]
    input: PathBuf,
```

or maybe even better `Option<PathBuf>` where `None` indicates that we shouldn't persist the orderbook. That way we do not have to pick a default path.

Then `UpdatingOrderbook::new` would also take and store `Option<PathBuf>`.

Reply:
Just want to mention that I have not implemented TryFrom for `PathBuf`. So I hope I can find some way to convert PathBuf into Path.

Reply:
+1 for the `Option<PathBuf>`.

Reply:
Done.